packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -210,7 +216,9 @@ export const ChannelView: React.FC = () => {
       <LimitedWidthContainer>
         {smMatch ? (
           <CollectorsBoxContainer>
-            {collectors.length > 0 && <CollectorsBox collectors={collectors} />}
+            {mappedChannelNftCollectors && mappedChannelNftCollectors.length > 0 && (
```

Comment:
```suggestion
            {mappedChannelNftCollectors.length > 0 && (
```
I think the result should be the same, especially if you apply my previous suggestion